magento/pwa-studio
# [PWA-2187]: [Bug] - Fix missing import statement on BraintreeSummary

## Description

BraintreeSummary component was crashing due to a missing import statement of `LoadingIndicator` and accessing  paymentNonce properties while being `null`  . This PR fixes these issues.
## Related Issue







Closes #3402 

## Acceptance





### Verification Stakeholders



### Specification




## Verification Steps






#### Test scenario(s) for direct fix/feature





1. Access checkout with braintree enabled
2. Fill out Shipping Information
3. Select a Shipping Method
4. Fill out braintree credit card details
5. Click on Review Order
6. Braintree Summary (Payment Information) should be rendered without any errors

#### Test scenario(s) for any existing impacted features/areas

Test same scenarios above in a scaffolded instance.





#### Test scenario(s) for any Magento Backend Supported Configurations





#### Is Browser/Device testing needed?





#### Any ad-hoc/edge case scenarios that need to be considered?




## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
